### PR TITLE
alert component

### PR DESCRIPTION
**Pull request by devAmlan**
This is an alert component.

### CODE REVIEW

File: library/component/alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.alert{
+    font-size:17px;
```
Comment (developer-anand) [inline library/component/alert/alert.css]: you can use rem here.

**Review by developer-anand — APPROVED** [review]
Good to go

File: library/component/alert/alert.css (hunk 1 above)
Comment (devAmlan) [reply library/component/alert/alert.css]: yeah, I will change this. thanks, anand.

**Review by sudipkundu999 — APPROVED** [review]
Looks good! Good to merge!
Just one suggestion to deploy the dev branch so that preview can be seen.

**Pull request merged**